server/src/index.ts

```diff
@@ -0,0 +1,49 @@
+import express from 'express';
+
+// import { ServerError } from '../src/types/types.ts';
+
+//! import middleware here:
```

Comment:
@LauraSchlueter22  Please delete this dead code. This is a great thing to add to our next iteration. But, it will be less of a conflict/less confusing if we start with a clean slate.

So, delete any comments that are not needed.